types/tabulator-tables/index.d.ts

```diff
@@ -1430,6 +1428,11 @@ export interface ColumnDefinition extends ColumnLayout, CellCallbacks {
     contextMenu?: Array<MenuObject<CellComponent> | MenuSeparator> | undefined;
     clickMenu?: Array<MenuObject<CellComponent> | MenuSeparator> | undefined;
 
+    /** Popups work in a similar way to menus, but instead of only displaying lists of menu items then allow you to fill them with any custom content you like, text, input elements, forms, anything you fancy. */
```

Comment:
typo: "... of only displaying lists of menu items then allow you to fill them ..."

should be "... of only displaying lists of menu items they allow you to fill them ..."

But this is probably coming from the tabulator webpage, right?

Reply:
Yeah I typically copy from the website documentation unless it's really bad.